packages/core/test/insured-bridge/e2e/helpers/ArtifactsHelper.js

```diff
@@ -0,0 +1,47 @@
+const hre = require("hardhat");
+
+const { findArtifactFromPath } = require("@uma/common");
+
+function getProviders() {
+  return {
+    l1RpcProvider: new hre.ethers.providers.JsonRpcProvider("http://localhost:9545"),
```

Comment:
Is there any way to get these urls from `HardhatConfig.js`? Just thinking ahead where we possibly change these ports and don't want unintended errors here.